sources/adjoint/adjoint_fluid_pnpn.f90

```
@@ -315,7 +312,14 @@ subroutine adjoint_fluid_pnpn_init(this, msh, lx, params, user, time_scheme)
     ! Select governing equations via associated residual and Ax types
     !
 
-    if (this%variable_material_properties .eqv. .true.) then
+    call json_get(params, 'case.numerics.time_order', integer_val)
+    allocate(this%ext_bdf)
+    call this%ext_bdf%init(integer_val)
+
+    call json_get_or_default(params, "case.fluid.full_stress_formulation", &
+         this%full_stress_formulation, .false.)
+
+    if (this%full_stress_formulation .eqv. .true.) then
```

hhhmmmm. Maybe we should throw an error here. Have we tested full stress + adjoint?